Lib/http/client.py

```diff
@@ -1377,6 +1377,8 @@ def __init__(self, host, port=None, key_file=None, cert_file=None,
             self.cert_file = cert_file
             if context is None:
                 context = ssl._create_default_https_context()
+                # send ALPN extension to indicate HTTP/1.1 protocol
```

Comment:
Are this comment and the functionality out of sync here since `http_vsn_str` can be `HTTP/1.0`, etc? `http/1.1`, `http/1.0`, and `http/0.9` are all valid ALPN protocol IDs but the comment and changelogs only mention 1.1. This might be fine anyways but something I noticed.

Reply:
I'm now checking ``_http_vsn`` and only send the ALPN extension when the version is ``11`` (``HTTP/1.1``).